src/emulator/host/include/host/state.h

```diff
@@ -53,10 +53,6 @@ struct DisplayState {
     std::condition_variable condvar;
     std::atomic<bool> abort{ false };
     std::atomic<bool> imgui_render{ true };
-
```

Comment:
this change should be in the other commit

Reply:
Done